llvm/test/CodeGen/AMDGPU/flat-scratch-ss-to-sv-scavenge.ll

```diff
@@ -0,0 +1,630 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; REQUIRES: amdgpu-registered-target
+; Ensure we don't crash with: "Cannot scavenge register in FI elimination!"
```

Comment:
Nit: It's better to put comments after the RUN lines, otherwise people might miss them.